src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ProcurementEventService.java

```diff
@@ -915,4 +915,42 @@ DocumentUpload findDocumentUploadInEvent(final ProcurementEvent event, final Str
             String.format(ERR_MSG_FMT_DOCUMENT_NOT_FOUND, documentId)));
   }
 
+  /**
+   * Export buyer attachments
+   * 
+   * @param procId
+   * @param eventId
+   * @param principal
+   * @return list of attachments
+   */
+  public List<DocumentAttachment> exportDocuments(final Integer procId, final String eventId,
+      final String principal) {
+    log.debug("Export all Documents from Event {}", eventId);
+    var event = validationService.validateProjectAndEventIds(procId, eventId);
+    var exportRfxResponse = jaggaerService.getRfx(event.getExternalEventId());
+    var status = jaggaerAPIConfig.getRfxStatusToTenderStatus()
+        .get(exportRfxResponse.getRfxSetting().getStatusCode());
+    var attachments = new ArrayList<DocumentAttachment>();
+
+    if (TenderStatus.ACTIVE != status) {
+      // Get documents from S3
+      event.getDocumentUploads().stream().forEach(doc -> {
+        var documentKey = DocumentKey.fromString(doc.getDocumentId());
+        var attachment = DocumentAttachment.builder()
+            .data(documentUploadService.retrieveDocument(doc, principal))
+            .fileName(documentKey.getFileName())
+            .contentType(MediaType.parseMediaType(doc.getMimetype())).build();
+        attachments.add(attachment);
```

Comment:
The unpublished path will need to invoke the draft document generation, according to ACs.  I can advise on that on Slack.